features/mbedtls/importer/adjust-check-config.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2019, Arm Limited, All Rights Reserved
```

Comment:
should have at least SPDX identifier

Reply:
It's on line 7 I think?
# SPDX-License-Identifier: Apache-2.0

Reply:
yes, now it is !